content/cookbook/01-next/121-send-pdf-to-llm.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Send a PDF to an LLM
```

Comment:
would rename the title as "Generate object with file prompt through form submission"